packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -424,6 +431,11 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 	 */
 	public isFixedHeaderEnabled = false;
 
+	/**
+	 * Whether the footer is enabled.
+	 */
+	public isFooterEnabled = false;
```

Comment:
So the value of this property should be disabled only if none of the columns define a footer...

Reply:
Ok, I'll get, I'll modify to get it from the column.